_posts/2021-05-14-status-update.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "2021-05-14: IAC & DevX Team Status Update"
+author: daianamezdrea
+categories:
+  - team
+  - status
+tags:
+- opv
+- EveryoneCanContribute
+- kubernetes 
+- motd
+---
+
+## Community Contributions
+
+We'd like to thank the following people in the Puppet Community for their contributions over this past week:
+
+- [`puppetlabs-kubernetes#516`][puppetlabs-kubernetes-pr-516]: "Add config.toml for containerd installed with 'archive'", thanks to [danifr][danifr]
+- [`puppetlabs-kubernetes#510`][puppetlabs-kubernetes-pr-510]: "Improvements to containerd configs when using a package", thanks to [treydock][treydock]
+- [`puppetlabs-postgresql#1272`][puppetlabs-postgresql-pr-1272]: "(MODULES-11069) add default version for fedora 34", thanks to [lweller][lweller]
+- [`puppetlabs-tomcat#446`][puppetlabs-tomcat-pr-446]: "Add service_name parameter to tomcat::instance", thanks to [treydock][treydock]
+- [`pdk-templates#432`][pdk-templates-pr-432]: "Fix Bug in .rubocop.yml.erb", thanks to [cocker-cc][cocker-cc]
+- [`pdk-templates#428`][pdk-templates-pr-428]: "Add EditorConfig", thanks to [jeffbyrnes][jeffbyrnes]
+- [`puppet-module-gems#163`][puppet-module-gems-pr-163]: "Pin docile on ruby 2.4", thanks to [alexjfisher][alexjfisher]
+
+Check [all the ways to reach us]({% link _posts/2021-01-20-reaching-out.md %}) if you want to directly contact us about anything module related.
+
+## New Module / Gem Releases
+
+The following modules were released this week:
+
+- [`puppetlabs-apache`][puppetlabs-apache] (`6.0.1`)
+- [`puppetlabs-motd`][puppetlabs-motd] (`6.0.0`)
+- [`puppetlabs-kubernetes`][puppetlabs-kubernetes] (`6.1.0`)
+- [`puppetlabs-iis`][puppetlabs-iis] (`8.0.2`)
+
+## Developer Experience
+
+The team has planned out the work for the next three weeks to complete a limited availability release of a new approach to templating with PDK. At the end of this we'll have an alpha release for people to try out and provide early feedback.
+We worked closely with the Puppet core team on some speed improvements over the last several months. Recent work has been merged into our Ruby distribution and into Puppet core to speed up all of our applications.
+Puppet VS Code extension work is coming along nicely, with all open PRs merged. We're planning on releasing 1.3.0 May 31st
+
+## Content Support Team
+
+A massive shout out to [Ben Ford](https://github.com/binford2k) for holding a group session this week and giving the team an overview of this tool. We were given plenty of opportunities to ask questions and further preparation for the handover of [dropsonde](https://github.com/puppetlabs/dropsonde). This is now on [Adrian][Adrian] radar and initial scoping of work continues. For more information on exactly how this can be used check out [Bens blog post](https://dev.to/binford2k/telemetry-that-doesn-t-suck-na2).
```

Comment:
```suggestion
A massive shout out to [Ben Ford](https://github.com/binford2k) for holding a group session this week and giving the team an overview of the [dropsonde](https://github.com/puppetlabs/dropsonde) tool. We were given plenty of opportunities to ask questions and further preparation for the handover of it to the IAC Team. This is now on [Adrian][Adrian]'s radar and initial scoping of work continues. For more information on exactly how this can be used check out [Bens blog post](https://dev.to/binford2k/telemetry-that-doesn-t-suck-na2).
```